### PR TITLE
fix(dashboard): Pass ALL_ACCESS_PROJECTS through to create/update dashboard requests

### PR DESCRIPTION
**Pull request by narsaynorath**
The backend has validation against querying for projects that are not actively selected, or do not exist. This passes down the constant for all projects when doing create/update actions since, for now, we don't save the dashboard page filters so we should not be requiring users to select projects to save their dashboards.

### CODE REVIEW

File: static/app/views/dashboardsV2/manage/dashboardList.tsx [hunk 1]
```diff
@@ -69,7 +71,9 @@ function DashboardList({
       const dashboardDetail = await fetchDashboard(api, organization.slug, dashboard.id);
       const newDashboard = cloneDashboard(dashboardDetail);
       newDashboard.widgets.map(widget => (widget.id = undefined));
-      await createDashboard(api, organization.slug, newDashboard, true);
+
+      // -1 for project IDs means it should include all projects
+      await createDashboard(api, organization.slug, newDashboard, true, [-1]);
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/manage/dashboardList.tsx:76]: should we reuse the const https://github.com/getsentry/sentry/blob/b519955596ed9d31f756f6a043721550f51d8263/static/app/constants/pageFilters.tsx#L26 here?

**Review by priscilawebdev — APPROVED** [review]
code looks good to me. Thanks for fixing it @narsaynorath 🌮

File: static/app/views/dashboardsV2/manage/dashboardList.tsx (hunk 1 above)
Comment (narsaynorath) [reply static/app/views/dashboardsV2/manage/dashboardList.tsx]: Great idea, I'm using it now, thanks!

**Review by wmak — APPROVED** [review]

File: .github/workflows/frontend.yml [hunk 2]
```diff
@@ -108,7 +108,8 @@ jobs:
       - name: stylelint (changed files only)
         if: github.ref != 'refs/heads/master' && needs.files-changed.outputs.frontend_components_modified_lintable == 'true'
         run: |
-          yarn stylelint ${{ needs.files-changed.outputs.frontend_components_modified_lintable_files }}
+          # XXX: this should use the set of changed files
+          yarn lint:css
```
Comment (asottile-sentry) [inline .github/workflows/frontend.yml:112]: I'll follow up on this tomorrow to do this properly

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**